specification/azsadmin/resource-manager/fabric/Microsoft.Fabric.Admin/preview/2016-05-01/ApplicationOperationResults.json

```diff
@@ -0,0 +1,188 @@
+{
+    "swagger": "2.0",
+    "info": {
+        "description": "Application operation results.",
+        "title": "FabricAdminClient",
+        "version": "2016-05-01"
+    },
+    "host": "adminmanagement.local.azurestack.external",
+    "schemes": [
+        "https"
+    ],
+    "consumes": [
+        "application/json"
+    ],
+    "produces": [
+        "application/json"
+    ],
+    "paths": {
+        "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Fabric.Admin/fabricLocations/{location}/applicationOperationResults/{operation}": {
+            "get": {
+                "x-ms-examples": {
+                    "Returns the status of the operation.": {
+                        "$ref": "./examples/ApplicationOperationResult/Get.json"
+                    }
+                },
+                "tags": [
+                    "ApplicationOperationResults"
+                ],
+            "description": "Returns the status of an application operation.",
+                "operationId": "ApplicationOperationResults_Get",
+                "parameters": [
+                    {
+                        "$ref": "Fabric.json#/parameters/SubscriptionIdParameter"
+                    },
+                    {
+                        "$ref": "Fabric.json#/parameters/ResourceGroupParameter"
+                    },
+                    {
+                        "$ref": "Fabric.json#/parameters/LocationParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/OperationParameter"
+                    },
+                    {
+                        "$ref": "Fabric.json#/parameters/ApiVersionParameter"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "OK",
+                        "schema": {
+                            "$ref": "#/definitions/ApplicationOperationResult"
+                        }
+                    },
+                    "404": {
```

Comment:
@sergey-shandar  Do you recommend having 404 error codes in swagger files? or they need to be removed? I believe if we keep the 404s here, the SDKs wont throw exception and silently eat the exception..

Reply:
I think its fine, we include 404s in all our admin swagger files..